peertube/.env-dist

```diff
@@ -0,0 +1,88 @@
+# The docker image to use:
+PEERTUBE_IMAGE=v6.1.0-bookworm
+# The Ppostgres image to use:
+PEERTUBE_POSTGRES_IMAGE=16-alpine
+
+# The domain name for the peertube service:
+PEERTUBE_TRAEFIK_HOST=peertube.example.com
+
+# The name of this instance. If there is only one instance, use 'default'.
+PEERTUBE_INSTANCE=
+
+# Filter access by IP address source range (CIDR):
+##Disallow all access: 0.0.0.0/32
+##Allow all access: 0.0.0.0/0
+PEERTUBE_IP_SOURCERANGE=0.0.0.0/0
+
+# HTTP Basic Authentication:
+# Use `make config` to fill this in properly, or set this to blank to disable.
+PEERTUBE_HTTP_AUTH=
+
+# OAUTH2
+# Set to `true` to use OpenID/OAuth2 authentication via the
+# traefik-forward-auth service in d.rymcg.tech.
+# Using OpenID/OAuth2 will require login to access your app,
+# but it will not affect what a successfully logged-in person can do in your
+# app. If your app has built-in authentication and can check the user
+# header that traefik-forward-auth sends, then your app can limit what the
+# logged-in person can do in the app. But if your app can't check the user
+# header, or if your app doesn't have built-in authentication at all, then
+# any person with an account on your Gitea server can log into your app and
+# have full access.
+PEERTUBE_OAUTH2=
+# In addition to Oauth2 authentication, you can configure basic authorization
+# by entering which authorization group can log into your app. You create
+# groups of email addresses in the `traefik` folder by running `make groups`. 
+PEERTUBE_OAUTH2_AUTHORIZED_GROUP=
+
+# Mutual TLS (mTLS):
+# Set true or false. If true, all clients must present a certificate signed by Step-CA:
+PEERTUBE_MTLS_AUTH=false
+# Enter a comma separated list of client domains allowed to connect via mTLS.
+# Wildcards are allowed and encouraged on a per-app basis:
+PEERTUBE_MTLS_AUTHORIZED_CERTS=*.clients.peertube.example.com
+
+
+## -- Peertube environment variables -- ##
+##--------------------------------------##
+# Database / Postgres service configuration
+PEERTUBE_POSTGRES_USER=postgres
+PEERTUBE_POSTGRES_PASSWORD=
+PEERTUBE_DB_SSL=false
+# Default to Postgres service name "postgres" in docker-compose.yml
+
+# PeerTube server configuration
+# If you need more than one IP as trust_proxy
+# pass them as a comma separated array:
+PEERTUBE_TRUST_PROXY=["127.0.0.1", "loopback", "172.18.0.0/16", "10.13.16.1/32"]
```

Comment:
we should probably leave this as just 127.0.0.1 (is loopback really necessary its the same right?) 

We can leave a commented out example for 10.13.16.1, but I'm not sure what 172.18.0.0 even is for.

Reply:
I don't know what these values should be, though I think 10.13.16.1 is correct for d.ry. The other values were in the source compose.yaml I started from, and I just left them - they can probably be deleted. 

Does this seem good?
```
# PeerTube server configuration
# If you need more than one IP as trust_proxy, pass them as a comma separated array.
# e.g., PEERTUBE_TRUST_PROXY=["127.0.0.1", "10.13.16.1/32"]
PEERTUBE_TRUST_PROXY=["127.0.0.1"]
```

Reply:
yea

Reply:
fixed in [117ae17](https://github.com/EnigmaCurry/d.rymcg.tech/pull/260/commits/117ae17d724b9ae0fc5115e157d9ebcd0fd34a88)